testsuite/tests/typing-gadts/name_existentials.ml

```diff
@@ -189,7 +189,7 @@ Line 2, characters 29-41:
                                  ^^^^^^^^^^^^
 Error: The local name "c" can only be given to an existential variable
        introduced by this GADT constructor.
-       The type annotation tries to bind it to the type "'a"
+       The type annotation tries to bind it to the type "_"
        that is not a locally abstract type.
```

Comment:
This changes highlights a small infelicity of the error message, it would be better to say "bind it to a type that is not a locally abstract type" rather than giving a name that does not come from the source. (It's pre-existing, not really a regression, although the change in printing makes it even more obvious.)